tests/subsys/pm/device_wakeup_api/src/main.c

```diff
@@ -9,12 +9,55 @@
 #include <device.h>
 #include <pm/pm.h>
 
+#define DT_DRV_COMPAT power_domain
+
 #define DEV_NAME DT_NODELABEL(gpio0)
 
 
-static const struct device *dev;
+static const struct device *gpio_dev, *domain_dev;
 static uint8_t sleep_count;
 
+static int domain_gpio_init(const struct device *dev)
```

Comment:
I'm not sure if mixing PM domain tests with wakeup API tests is a good idea.

Reply:
removed !